lua/cmp/entry.lua

```diff
@@ -339,7 +339,9 @@ entry.get_replace_range = function(self)
       else
         replace_range = self:get_completion_item().textEdit.range
       end
-    else
+    end
+
+    if not replace_range or (self.context.cursor.character == replace_range['end'].character) then
```

Comment:
I understood this to mean that if the obtained replace-range looks like an insert-range, it creates a replace-range with its own logic. Is this correct?

Reply:
> obtained replace-range looks like an insert-range

Not sure tbh, but the second part is correct.